G2P.pm

```diff
@@ -230,6 +230,14 @@ sub new {
     die("ERROR: Couldn't create log_dir $log_dir $!\n") if (!$return);
     $params->{log_dir} = $log_dir;
   } 
+  else{
+    opendir my $dh, $log_dir or warn "Not a directory";
+    my @check = grep {$_ ne '.' and $_ ne '..'} readdir $dh;
+    die("Error: Directory is not empty, You need to empty directory before using the plugin \n") if (scalar @check != 0);
```

Comment:
Could you please use ERROR instead of Error for consistency across the plugin. And could you please update the message to: ERROR: The log directory ($log_dir) is not empty. You need to empty the directory before using the plugin.